### PR TITLE
Increase table width

### PR DESCRIPTION
**Pull request by ElleeB**
Increase table width on additional pages.

### CODE REVIEW

File: modules/appeals_api/app/services/appeals_api/pdf_construction/notice_of_disagreement/pages/hearing_type_and_additional_issues.rb [hunk 1]
```diff
@@ -22,7 +22,7 @@ def build!
             return pdf unless extra_issues?
 
             pdf.text("\n<b>Additional Issues</b>\n", inline_format: true)
-            pdf.table(extra_issues_table_data, header: true)
+            pdf.table(extra_issues_table_data, width: 540, header: true)
```
Comment (yakschuss) [inline modules/appeals_api/app/services/appeals_api/pdf_construction/notice_of_disagreement/pages/hearing_type_and_additional_issues.rb:25]: What is the value 540 based off of?

Comment (ElleeB) [reply modules/appeals_api/app/services/appeals_api/pdf_construction/notice_of_disagreement/pages/hearing_type_and_additional_issues.rb:25]: 540 is the number of pixels that created an even margin on either side of the table, and left it just smaller than the margins of the table/columns of text above.

**Comment by ElleeB** [comment]
Screen Shot 2020-12-28 at 2 15 56 PM

**Comment by ElleeB** [comment]
Screen Shot 2021-01-19 at 11 58 27 AM

**Review by yakschuss — APPROVED** [review]

**Pull request merged**